cmake/SetupDoxygen.cmake

```diff
@@ -140,6 +140,8 @@ ${DOCS_POST_PROCESS_COMMAND} && exit \${generate_docs_exit}\n"
       --remove ${CMAKE_BINARY_DIR}/docs/tmp/doc_coverage.info
       '${CMAKE_SOURCE_DIR}/src/Executables/*'
       '${CMAKE_SOURCE_DIR}/tests/*'
+      '${CMAKE_SOURCE_DIR}/citelist'
+      '${CMAKE_SOURCE_DIR}/[generated]'
```

Comment:
I don't understand what you are trying to match with this line (i.e. why use [] ?)

Reply:
This literal line appears in the `doc_coverage.info` for some reason, so I remove it 🤷‍♂️

Reply:
Does this work?  It's not clear what sort of interpretation is being done on this, but `[]` has special meaning to the shell.

Reply:
It works and fixes docs deployment on develop.

Reply:
okay, it appears lcov and I have a difference of opinion on what constitutes a shell wildcard pattern...

Reply:
This is probably because of the weird feature that a pattern that matches nothing expands to itself.  Single character filenames aren't likely to occur here, so I think this is fine.  (Particularly as it is not clear how to escape things here.)